soiree/app/controllers/group/playlists_controller.rb

```diff
@@ -29,6 +29,21 @@ def delete_group_redirect(playlist)
     redirect_to dashboard_path
   end
 
+  def delete_group_invites(playlist)
+    #When a group member deletes their portion of the group playlist, their image should be taken off other group
+    # Members pages
+    binding.pry
+    group_id = playlist.group_playlists[0].group_id
+
+    if GroupUser.where(group: group_id).length == 1
+      # Determine if there is only one person in the group
+      # Iterate through and delete the invites if any
+      # else
+      # Iterate through and delete the user from the group playlist
```

Comment:
It looks like the Invite model has a group foreign key: 

class Invite < ActiveRecord::Base
  belongs_to :user
  belongs_to :group

are you throwing an error with destroy, or is it just not deleting the invites?